### PR TITLE
Tests for StaticMap, LatLng. Clean up markdown warnings in README.md.

### PR DESCRIPTION
**Pull request by domesticmouse**
Also, in passing, updated the StaticMap implementation to return a decoded image, instead of the raw HTTP Response. Felt more appropriate.

### CODE REVIEW

File: staticmap.go [hunk 1]
```diff
@@ -256,12 +275,29 @@ func (r *StaticMapRequest) params() url.Values {
...
-	return c.get(ctx, staticMapAPI, r)
+
+	resp, err := c.get(ctx, staticMapAPI, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != 200 {
+		if b, err := ioutil.ReadAll(resp.Body); err == nil {
+			return nil, fmt.Errorf("Static Map API returned: %s", string(b))
```
Comment (samthor) [inline staticmap.go]: this doesn't include the status code, if say the body doesn't: maybe include it?

File: staticmap.go [hunk 2]
```diff
@@ -256,12 +275,29 @@ func (r *StaticMapRequest) params() url.Values {
...
+		}
+		return nil, err
+	}
+
+	img, _, err := image.Decode(resp.Body)
+	if err != nil {
```
Comment (samthor) [inline staticmap.go]: You can just replace the tail with
`return img, err`

File: staticmap.go (hunk 1 above)
Comment (domesticmouse) [inline staticmap.go]: Done

File: staticmap.go (hunk 2 above)
Comment (domesticmouse) [inline staticmap.go]: Done

**Pull request merged**